src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 :nth-child(-n + 4){
```

Comment:
no need to use all variants (from 0 to 5) in HTML, but it's better to keep them in CSS to make them work if it'll be needed
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/e91dcdba-f96a-45fb-b644-2591b0a463c0)